main.py

```diff
@@ -0,0 +1,70 @@
+"""
+You could download multiple courses at once 
+with help of the script from stepic.org .
+
+
+Run by: 
+python3 main.py
+
+Additional notes:
+1) Add courses which you want to download to the "courses" dictionary.
+
+2) You could also set quality of the video by changing "quality" variable. 
+The default quality is 720.
+"""
+
+import os
+
+import weekDownloader
+
+def main():
```

Comment:
Another main function? So, which one should I run now? This, or main from weekDownloader?
It's always better to have one entry point for script